ios-competency-test/ios-competency-test/FriendCommand.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  FriendCommand.swift
+//  ios-competency-test
+//
+//  Created by Sam Gubler on 3/27/15.
+//  Copyright (c) 2015 Sam Gubler. All rights reserved.
+//
+
+import UIKit
+import Alamofire
```

Comment:
Did you actually use Alamofire? If so how did you add it to the project? Cocoapod? Framework?